src/Types.ts

```diff
@@ -826,6 +942,16 @@ export interface ITxStatus {
     };
 }
 
+/**
+ * Define the types of ballot anwser 
+ */
+export enum BallotAnswer {
+    Yes = 0,
+    No = 1,
+    Blank = 2,
+    Reject = 9,
+}
```

Comment:
https://github.com/bosagora/boa-sdk-ts/blob/a7e1bff6029d5dc3792887bbe373c3b55afbe4fa/src/modules/vote/BallotData.ts#L148-L151

Reply:
This is for frontend, since we have to create array in order to check the ballot answer